include/edm4hep/Constants.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2014-2023 Key4hep-Project.
+ *
+ * This file is part of Key4hep.
+ * See https://key4hep.github.io/key4hep-doc/ for further info.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef EDM4HEP_CONSTANTS_H
+#define EDM4HEP_CONSTANTS_H
+
+#include <string>
+
+namespace edm4hep {
+  inline constexpr std::string_view CellIDEncoding = "CellIDEncoding";
```

Comment:
```suggestion
  inline static constexpr auto CellIDEncoding = "CellIDEncoding";
```
I think making this a `static const char*` makes it easier to consume for things that take a `const std::string&` as  a `string_view` will lead to

```
error: invalid initialization of reference of type 'const std::string&' {aka 'const std::__cxx11::basic_string<char>&'} from expression of type 'const std::string_view' {aka 'const std::basic_string_view<char>'
```
[compiler explorer example](https://godbolt.org/z/sWaKo5vzY)

Reply:
This will also still work if functions take a `string_view` as argument.

Reply:
I made it `const char*`; `inline` can be dropped since it's static